src/modules/governance/services/governance.abi.service.ts

```diff
@@ -452,3 +453,93 @@ export class GovernanceEnergyAbiService
             .toPlainObject();
     }
 }
+
+@Injectable()
+export class GovernanceOldEnergyAbiService extends GovernanceEnergyAbiService {
+    constructor(
+        protected readonly mxProxy: MXProxyService,
+        protected readonly governanceMerkle: GovernanceTokenSnapshotMerkleService,
+        protected readonly governanceDescription: GovernanceDescriptionService,
+    ) {
+        super(mxProxy, governanceMerkle, governanceDescription);
+        this.type = GovernanceType.OLD_ENERGY;
+    }
+
+
+    async energyFactoryAddress(scAddress: string): Promise<string> {
+        return Promise.resolve('');
+    }
+
+    async feeTokenId(scAddress: string): Promise<string> {
+        return Promise.resolve('LKMEX-aab910');
```

Comment:
I would not hardcode this value. Use the locked asset factory abi to get this value

Reply:
done